pointcloud_clustering/positionRPY odometry
geometry_msgs/PoseArray verticalElements
geometry_msgs/PoseArray verticalElements_BL
---
pointcloud_clustering/positionRPY corrected_position